gitlens/gitlens-features.md

```diff
@@ -633,3 +639,15 @@ GitLens supports [user-defined](#modes-settings- 'Jump to the Modes settings') m
 - Adds a _Zen_ mode which for a zen-like experience, disables many visual features
   - Adds _Toggle Zen Mode_ command (`gitlens.toggleZenMode`) to toggle Zen mode
 - Adds a _Review_ mode which for reviewing code, enables many visual features
+
+### Experimental Multi-diff Editor
```

Comment:
@d13 @axosoft-ramint two questions about this feature:
- Should this have its own section on the help center or since it is part of a VS Code feature, should the release notes be enough to cover this?
- If it should have its own section, is the GitLens Feature page the correct place?

Reply:
I'm on the fence on this one. On the one hand it's good to expose that GitLens can leverage VS Code's multi-diff editor. On the other hand, the multi-diff part is VS Code's thing, not ours - we are just pointing commands at it.

We seem to be pretty clear about that distinction in the wording used here, so I'm ok with this having a help center section if it helps users discover it.